src/components/InputDropdown/index.js

```diff
@@ -1,80 +1,60 @@
 /** @jsx jsx */
-import { jsx } from '@emotion/core'
-import React from 'react'
-import Text from 'components/Text'
-import { ReactComponent as Chevron } from 'static/icons/chevron.svg'
+import { jsx } from '@emotion/core';
+import React from 'react';
+import { ReactComponent as Chevron } from 'static/icons/chevron.svg';
 
-import styles from './InputDropdown.styles'
+import styles from './InputDropdown.styles';
+
+const DEFAULT = 'placeholder';
 
 class InputDropdown extends React.Component {
   constructor(props) {
-    super(props)
-    this.listEl = null
-    this.inputEl = null
+    super(props);
     this.state = {
-      isOpen: false,
-      value: '',
-    }
-  }
+      value: DEFAULT,
+    };
 
-  componentDidMount() {
-    window.addEventListener('mousedown', this.handleClick)
+    this.handleChange = this.handleChange.bind(this);
   }
 
-  componentWillUnmount() {
-    window.removeEventListener('mousedown', this.handleClick)
-  }
+  handleChange = (e) => {
+    const { value: newValue } = e.target;
 
-  handleClick = (e) => {
-    this.setState((prevState) => {
-      // clicked outside list
-      if (
-        (prevState.isOpen && !this.listEl.contains(e.target)) ||
-        e.target === this.inputEl
-      ) {
-        return { isOpen: !this.state.isOpen }
-      }
-    })
-  }
-
-  onChange = (option) => {
     if (this.props.customOnChange) {
-      this.props.customOnChange(option.value)
+      this.props.customOnChange(newValue);
     }
-    this.setState({ value: option.label, isOpen: false })
-  }
+    this.setState({ value: newValue });
+  };
 
   render() {
-    const { isOpen, value } = this.state
-    const { placeholder, options = [] } = this.props
+    const { placeholder, options = [] } = this.props;
 
     return (
-      <div css={[styles.root, isOpen && styles.active]}>
-        <span css={[styles.placeholder, value && styles.selected]}>
-          <Text>{value || placeholder}</Text>
-        </span>
-        <div
-          value={value}
-          ref={(ref) => (this.inputEl = ref)}
-          css={styles.dropdown}>
-          {isOpen && (
-            <ul css={styles.list} ref={(ref) => (this.listEl = ref)}>
-              {options.map((option) => (
-                <li
-                  onClick={() => this.onChange(option)}
-                  css={styles.option}
-                  key={option.label}
-                  value={option.value}>
-                  <Text>{option.label}</Text>
-                </li>
-              ))}
-            </ul>
-          )}
-        </div>
-        <Chevron css={[styles.chevron, isOpen && styles.chevronOpen]} />
+      <div css={styles.root}>
+        <select
+          css={[
+            styles.select,
+            this.state.value === DEFAULT && styles.selectDefaultState,
+          ]}
+          onChange={this.handleChange}
+          value={this.state.value}
+        >
```

Comment:
```suggestion
          {...rest}
        >
```

Reply:
@joshrounsville has a good point that this can be error prone... do you think it would be better to have component specific props, like `inputProps` and `selectProps` so we can spread those instead of all other props?

Reply:
I'm actually in favor of explicit props, so we'd have `name` in this case.

I just suggested based on what our other inputs do.

Reply:
now could be a good time to slowly begin refactoring that to accept those explicit props in that case! the `rest` paradigm was intro'd during the "get it done in 4 days timeline"

Reply:
`inputProps` it is!